### PR TITLE
Add lodewallet

### PR DESCRIPTION
**Pull request by AYWMM7**
## Description
Added lodewallet to the wallets list

## Where should the reviewer start?
"Transparent support" was marked as :x: by default.  I don't know what this parameter means. It would be ideal if a reviewer could check this is correct.  I can help answer questions, or fill this field if I am told what it means.

## Motivation and context
Lodewallet is the only native Haskell light wallet for Cardano. It makes direct use of open-source Cardano Haskell libraries.

### CODE REVIEW

**Comment by rdlrt** [comment]
> I don't know what this parameter means.

That's a good point, it's not described on the site, will need to describe it.

Essentially `Transparent Support` is ensuring there are channels/folks/platform that can be approached where users can see already asked queries and replies (often some wallets outsource their support to third party websites, or have their support only via emails) - where unidirectional comms platforms like twitter do not really qualify. For cryptocurrency wallets, this often becomes a source of frustration for users , as they can end up in uncertain timelines for resolution, or they cannot easily assess if it's something they're impacted themselves or if it has already been asked/raised.

File: docs/Wallets/list.md [hunk 1]
```diff
@@ -11,20 +11,21 @@ Note that in case of issues, usually only queries relating to official wallets c
...
+|[Nami]       |Light    |7          |:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |( :x:,:ballot_box_with_check:,:x: )                     |[:ballot_box_with_check:](https://github.com/Berry-Pool/nami-wallet)|
+|[Gero]       |Light    |7          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:x:                    |( :x:,:ballot_box_with_check:,:x: )                     |:x:                    |
+|[LodeWallet] |Light    |6          |:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:x:                    |:x:                    |:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|( :x:,:ballot_box_with_check:,:x: )                     |:x:                    |
```
Comment (rdlrt) [inline docs/Wallets/list.md:25]: looks like there was accidentally a carriage return instead of being on single line

Comment (AYWMM7) [reply docs/Wallets/list.md:25]: What do you mean?  All the lodewallet info seems to be on line 25.  And grepping for `\r`brings up nothing, so this file shouldn't have any carriage returns.  I must be misunderstanding.

Comment (rdlrt) [reply docs/Wallets/list.md:25]: You're right, my bad - copying the line messed up in mobile, as it was showing up wrapped text due to next fields...false alarm

**Review by rdlrt — APPROVED** [review]

**Pull request merged**